lib/spack/spack/test/cmd/buildcache.py

```diff
@@ -57,6 +58,7 @@ def test_buildcache_preview_just_runs(database):
     buildcache('preview', 'mpileaks')
 
 
+@pytest.mark.skipif(sys.platform == 'win32', reason="Not yet implemented on windows")
```

Comment:
This looks like another instance where each test in the module is skipped: can you do the module-level skip (and remove individual skips) in each module where this is the case?